tests/edit_inputs.sh

```diff
@@ -42,13 +38,14 @@ function edit_mom_input {
       -e "s/MOM6_RIVER_RUNOFF/$MOM6_RIVER_RUNOFF/g" \
       -e "s/MOM6_THERMO_SPAN/$MOM6_THERMO_SPAN/g" \
       -e "s/MOM6_REPRO_LA/$MOM6_REPRO_LA/g" \
+      -e "s/MOM6_USE_WAVES/$MOM6_USE_WAVES/g" \
       -e "s/NX_GLB/$NX_GLB/g" \
       -e "s/NY_GLB/$NY_GLB/g" \
-      -e "s/CHLCLIM/$CHLCLIM/g"
+      -e "s/CHLCLIM/$CHLCLIM/g" 
```

Comment:
Remove whitespace

Reply:
I will follow Jun's suggestion that I make a PR to your upcoming PR #238 to remove the whitespace.